backend/config/db.js

```diff
@@ -1,18 +1,24 @@
 import mysql from 'mysql2';
-import dbConfig from './db.config.js';
+import dotenv from 'dotenv';
+
+dotenv.config()
```

Comment:
this config init should be in the main file. ENV vars do not relates only to the DB, might be any else